.env.defaults

```
@@ -34,10 +34,15 @@ BUCKET_EXPIRATION_IN_DAYS=1
 LOG_LOCATION=./logs
 LOG_FILES_EXPIRATION_IN_DAYS=2
 
-########################## CxFlow #################################
+######################### CxSCA Credentials #######################
 SCA_TENANT=tenant
 SCA_USERNAME=sca_username
 SCA_PASSWORD=sca_password
+
+######################### Xray Booster ############################
+API_AUTH_TOKEN=1234
```

if this is something static, that we supply, then we can do the same we do for minio

```suggestion
API_AUTH_TOKEN=%AUTO_GENERATED_PASSWORD%
```